pkg/pillar/cmd/zedrouter/networkinstance.go

```diff
@@ -1239,6 +1239,12 @@ func doNetworkInstanceActivate(ctx *zedrouterContext,
 	if status.Type == types.NetworkInstanceTypeSwitch {
 		// switched NI is not probed and does not have a CurrentUplinkIntf
 		status.IfNameList = getIfNameListForLLOrIfname(ctx, status.Logicallabel)
+		if len(status.IfNameList) > 1 {
+			err := fmt.Errorf("Name %s maps to more than one (%d) interfaces",
```

Comment:
Should the error be instead `Network instance %s ...` - I mean to make it clear for the user that this error is for object of type "network instance".

Reply:
I'll make the log include that string. The error itself will also be set in the NetworkInstanceStatus and no need to repeat the NI name there.

Reply:
OK, make sense